notebooks/en/issues_in_text_dataset.ipynb

```diff
@@ -0,0 +1,571 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    label_issues.head() 
</pre><p>let's add the output. When running this code, why is the output from the report different from this? The label issues I get from the lab.report are: "Examples representing most severe instances of this issue:</p><p>     is_label_issue  label_score              given_label           predicted_label</p><p>981            True     0.000005     card_about_to_expire  card_payment_fee_charged</p><p>974            True     0.000150  beneficiary_not_allowed                change_pin</p><p>982            True     0.000218  apple_pay_or_google_pay      card_about_to_expire</p><p>971            True     0.000512  beneficiary_not_allowed                change_pin</p><p>980            True     0.000947     card_about_to_expire  card_payment_fee_charged</p><p>"</p><p> </p><p>These makes sense, however, label_issues.head() returns a different set of examples: </p><p><code>   is_label_issue  label_score      given_label          predicted_label</p><p>0           False     0.792001  cancel_transfer          cancel_transfer</p><p>1           False     0.258453  cancel_transfer          cancel_transfer</p><p>2           False     0.698996  cancel_transfer          cancel_transfer</p><p>3           False     0.183092  cancel_transfer  apple_pay_or_google_pay</p><p>4           False     0.771016  cancel_transfer          cancel_transfer</code></p><p> </p><p>Why are these different? Why is it a label issue if the predicted label matches the given label?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Reply:
<p>It is just sorted differently, than the report functionality which is more articulate and aggregate of all the issues in the dataset. </p><p> </p><blockquote>Why is it a label issue if the predicted label matches the given label?  </blockquote><p>Primarily, whether a data point is having a labelling issue or not is given by the <code>is_label_issue</code> column. On top the probabilities indicate the possibility it being a issue.  Unless the <code>is_label_issue</code> is <code>True</code> , it is not marked as a problem in the dataset. </p><p> </p><p>For all the data points, the metrics are generated so that user can take a look!</p>